src/main/kotlin/step1/Person.kt

```diff
@@ -0,0 +1,7 @@
+package step1
```

Comment:
In this mission, we're creating a simple car racing program. Instead of building packages step by step, we'll continuously refactor from one package to the next. With that in mind, why not give the `step1` package a more meaningful name as well?